server/bundler/utils.js

```diff
@@ -23,7 +31,44 @@ function pathToRegExp( path ) {
 	return new RegExp( '^' + path + '(/.*)?$' );
 }
 
+function hashFile( path ) {
+	const md5 = crypto.createHash( 'md5' );
+	let data, hash;
+
+	try {
+		data = fs.readFileSync( path );
+		md5.update( data );
+		hash = md5.digest( 'hex' );
+		hash = hash.slice( 0, HASH_LENGTH );
+	} catch ( e ) {
+		hash = new Date().getTime().toString();
+	}
+
+	return hash;
+}
+
+function getUrl( filename, hash ) {
+	return URL_BASE_PATH + '/' + filename + '?' + qs.stringify( {
+		v: hash
+	} );
+}
+
+function getHashedUrl( filename ) {
+	return getUrl( filename, hashFile( process.cwd() + SERVER_BASE_PATH + '/' + filename ) );
+}
+
+function getCssUrls( css ) {
+	return {
+		ltr: getHashedUrl( 'sections/' + css + '.css' ),
+		rtl: getHashedUrl( 'sections-rtl/' + css + '.rtl.css' ),
+	};
+}
+
 module.exports = {
 	getAssets: getAssets,
-	pathToRegExp: pathToRegExp
+	pathToRegExp: pathToRegExp,
+	hashFile: hashFile,
+	getUrl: getUrl,
+	getHashedUrl: getHashedUrl,
```

Comment:
Do we really need to export `getHashedUrl()`? As far I can see it's never used outside of this file. We can keep that function private for now.